vprdb/converters/kitti_converter.py

```diff
@@ -0,0 +1,71 @@
+import open3d as o3d
+import pykitti
+import numpy as np
+import mrob
+
+
+def kitti_to_lib_format(
+    source_dir: str,
+    sequence: str,
+    camera: int,
+    output_pcd_dir: str,
+    output_poses_dir: str,
+):
+    if camera != 2 and camera != 3:
+        raise ValueError("Invalid camera choice. Acceptable values are '2' or '3'.")
+
+    dataset = pykitti.odometry(source_dir, sequence)
+
+    if camera == 2:
+        cam_to_velo = dataset.calib.T_cam2_velo
+        cam_intrinsics = dataset.calib.K_cam2
+    else:
+        cam_to_velo = dataset.calib.T_cam3_velo
+        cam_intrinsics = dataset.calib.K_cam3
+
+    poses = dataset.poses
+    poses_to_write = []
```

Comment:
It seems that you don't use it.
But I would leave it and write trajectory to the file after `for` loop. Then you don't need to wrap the whole loop with `with` statement